modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -262,182 +284,160 @@ Protobuf::
-iceberg_rest_catalog_client_id: <rest-connection-user>
-iceberg_rest_catalog_client_secret: <rest-connection-password>
+{
+  "type": "record",
+  "name": "ClickEvent",
+  "fields": [
+    {
+      "name": "user_id",
+      "type": "int"
+    },
+    {
+      "name": "event_type",
+      "type": "string"
+    }.
+    {
+      "name": "ts",
+      "type": [
+          "null",  
+          { "type": "string", "logicalType": "date" }
+        ],
+      "default": null  # Default value for the new field
+    }
+  ]
+}
 ----
+The `ts` field can be either null or a string representing a date. The default value is null.
 
-And you use Spark as a processing engine, configured to use the `streaming` catalog:
+[,bash]
+----
+rpk registry schema create demo-topic-value --schema schema_2.avsc
 
-```
-spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
-spark.sql.catalog.streaming.type = rest
-spark.sql.catalog.streaming.uri = http://catalog-service:8181
-```
+echo '{"user_id":858, "event_type":"BUTTON_CLICK", "ts":{"string":"2025-02-26T20:05:23.230ZZ"}}' | rpk topic produce demo-topic --format='%v\n' --schema-id=topic
+----
 
-Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+xref:manage:iceberg/query-iceberg-topics.adoc[Querying the Iceberg table] for `demo-topic` includes the new column `ts`:
 
-[,sql]
+[,bash,role=no-copy]
 ----
-SELECT * FROM streaming.redpanda.ClickEvent;
++---------+--------------+--------------------------+
+| user_id | event_type   | ts                       |
++---------+--------------+--------------------------+
+| 858     | BUTTON_CLICK | 2025-02-26T20:05:23.230Z |
+| 23      | BUTTON_CLICK | NULL                     |
++---------+--------------+--------------------------+
 ----
 
-Spark can use the REST catalog to automatically discover the topic's Iceberg table. 
+== Manage dead-letter queue
 
-See also: xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[]
+Errors may occur when translating records in the `value_schema_id_prefix` mode to the Iceberg table format; for example, if you do not use the Schema Registry wire format with the magic byte, if the schema ID in the record is not found in the Schema Registry, or if an Avro or Protobuf data type cannot be translated to an Iceberg type.
 
-==== File system-based catalog (`object_storage`)
+If Redpanda encounters an error while writing a record to the Iceberg table, Redpanda writes the record to a separate dead-letter queue (DLQ) Iceberg table named `<topic-name>~dlq`. To disable the default behavior for a topic and drop the record, set the xref:reference:properties/topic-properties.adoc#redpandaiceberginvalidrecordaction[`redpanda.iceberg.invalid.record.action`] topic property to `drop`. You can also configure the default cluster-wide behavior for invalid records by setting the config_ref:iceberg_invalid_record_action,true,properties/cluster-properties[`iceberg_invalid_record_action`] property.
```

Comment:
Please check link for iceberg_invalid_record_action property.

Reply:
This link will work when the RC is available and we can extract the cluster config property reference/